src/mocks.ts

```diff
@@ -32,6 +34,8 @@ export const mockManifest = () => {
     'cache.html': `cache.${HASH}.html`,
     'index.html': `index.${HASH}.html`,
     'sub/blah.png': `sub/blah.${HASH}.png`,
+    'client': `client.${HASH}`,
+    'client/index.html': `client.${HASH}`,
```

Comment:
it could be really good to add a test for `client/` to make sure that retrieves whatever is at `client/index.html`!

Reply:
This is mocking what we store in KV, so there would never be a trailing slash even for those requests with a trailing slash. There is a test for trailing slash though (src/test/getAssetFromKV.ts: 20)